### PR TITLE
Reworked command interface, added to help command

### PR DESCRIPTION
**Pull request by Mattihew**

### CODE REVIEW

File: src/main/java/com/logdyn/ui/console/ConsoleApplication.java [hunk 1]
```diff
@@ -52,10 +53,14 @@ private static void interactive() {
     }
 
     private static void execute(final String[] args) {
-        commands.stream()
-                .filter(command -> command.getName().equalsIgnoreCase(args[0]))
-                .findAny()
-                .orElse(HELP_COMMAND)
-                .execute(Arrays.copyOfRange(args, 1, args.length));
+        if (args.length > 1 && HELP_COMMAND.getNames().contains(args[1].toLowerCase())) {
```
Comment (jsjlewis96) [inline src/main/java/com/logdyn/ui/console/ConsoleApplication.java]: Maybe extract to boolean var to explain what it's checking for, I can't easily tell

File: src/main/java/com/logdyn/ui/console/commands/HelpCommand.java [hunk 2]
```diff
@@ -19,6 +21,31 @@ public HelpCommand(final Collection<Command> commands) {
...
+            }
+            else
+            {
+                DefaultCommand.execute(args[0]);
+            }
+        }
+    }
+
+    private void printHelp(final Command command){
+        System.out.printf("[%s] %s - %s%n", String.join(" | ", command.getNames()), command.getDescription(), command.getHelp());
```
Comment (jsjlewis96) [inline src/main/java/com/logdyn/ui/console/commands/HelpCommand.java]: Could maybe use a comment to show an example output

File: src/main/java/com/logdyn/ui/console/commands/HelpCommand.java [hunk 3]
```diff
@@ -32,12 +59,22 @@ public void execute(final String... args) {
...
-            System.out.println(String.format("Help page can be found at %s", HELP_URL));
+            System.out.printf("Help page can be found at %s%n", HELP_URL);
         }
     }
 
     @Override
-    public String getName() {
-        return "help";
+    public Collection<String> getNames() {
+        return Arrays.asList("help", "--help", "/?");
```
Comment (jsjlewis96) [inline src/main/java/com/logdyn/ui/console/commands/HelpCommand.java]: Still not keen on /? . It's the only flag in that style, and it doesn't work in all shells.

File: src/main/java/com/logdyn/ui/console/commands/HelpCommand.java (hunk 2 above)
Comment (Mattihew) [reply src/main/java/com/logdyn/ui/console/commands/HelpCommand.java]: done

File: src/main/java/com/logdyn/ui/console/commands/HelpCommand.java (hunk 3 above)
Comment (Mattihew) [reply src/main/java/com/logdyn/ui/console/commands/HelpCommand.java]: tough

**Pull request merged**